frontend/apps/marketing/src/components/contentful/skinnyBanner/SkinnyBannerContentfulDefinition.ts

```diff
@@ -0,0 +1,92 @@
+import {ComponentDefinition} from '@contentful/experiences-sdk-react';
+
+export const SkinnyBannerContentfulComponentDefinition: ComponentDefinition = {
+  id: 'skinnyBanner',
+  name: 'Skinny Banner',
+  category: '01: Page Structure',
```

Comment:
This should be here:
```suggestion
  category: '04: Advanced',
```